newIDE/app/src/ObjectEditor/ObjectEditorDialog.js

```diff
@@ -66,8 +66,17 @@ const InnerDialog = (props: InnerDialogProps) => {
 
   const EditorComponent = props.editorComponent;
 
+  const _onApply = () => {
```

Comment:
nitpicking: replace the `const _onApply` by `const onApply`.
Underscores are just for "private" functions in a class. Otherwise, just use `camelCase` for naming your functions and variables.